dpypelines/pipeline/dataset_ingress_v1.py

```diff
@@ -211,15 +210,16 @@ def dataset_ingress_v1(files_dir: str, pipeline_config: dict):
                     submitter_email, email_content.subject, email_content.message
                 )
                 logger.info(
-                    "Sent email to submitter: ",
-                    submitter_email,
-                    " about missing supplementary distribution: ",
-                    supp_dist_pattern,
+                    "Email sent to submitter about missing supplementary distribution file",
+                    data={
+                        "submitter_email": submitter_email,
+                        "supplementary_distribution_pattern": supp_dist_pattern,
+                    },
                 )
                 de_notifier.failure()
```

Comment:
wasn't you but but there's a logic error here. A missing file is an error not information, we need the pipeline to error and stop processing.

Can you (a) create an error (b) log it, then (c) raise it after we notify of the failure please.

you can just instantiate an error manually, it wont raise in of itself, something like:

```
err = FileNotFoundError(something useful>)
```